core/state_transition.go

```diff
@@ -339,9 +345,70 @@ func (st *StateTransition) TransitionDb(refunds bool, gasBailout bool) (*evmtype
 	rules := st.evm.ChainRules()
 	vmConfig := st.evm.Config()
 	isEIP3860 := vmConfig.HasEip3860(rules)
+	accessTuples := make(types2.AccessList, 0)
+	if msg.AccessList() != nil {
+		accessTuples = append(accessTuples, msg.AccessList()...)
```

Comment:
Looks equal to `slices.Clone()`

Reply:
yes, will remove it

Reply:
oops. Auto-merge was on...will put this in a new pr.